gym/utils/step_api_compatibility.py

```diff
@@ -128,33 +129,33 @@ def step_to_old_api(
 
 
 def step_api_compatibility(
-    step_returns: Union[NewStepType, OldStepType],
-    new_step_api: bool = False,
+    step_returns: Union[TerminatedTruncatedStepType, DoneStepType],
+    output_truncation_bool: bool = True,
```

Comment:
I don't like this argument name, it's pretty unclear. I think there's a different name used earlier?

Reply:
`to_termination_truncation_api` ? I agree it is not a great name but uncertain of a better one